apis/python/src/tiledbsoma/_sparse_nd_array.py

```diff
@@ -528,7 +528,7 @@ def _dim_capacity_and_extent(
 
     def used_shape(self) -> Tuple[Tuple[int, int], ...]:
         """
-        Retrieve the range of indexes for a dimension that were explicitly written.
+        Retrieve the range of indexes for a dimension that were explicitly written (deprecated).
```

Comment:
I found that sphinx really likes to take only the first sentence and use it in various places for summaries. The deprecation status of this is important, and it belongs in that first sentence.